2.4/Dockerfile.fedora

```diff
@@ -1,15 +1,14 @@
 FROM registry.fedoraproject.org/f27/s2i-base:latest
 
-# This image provides a Ruby 2.4 environment you can use to run your Ruby
+# This image provides a Ruby environment you can use to run your Ruby
 # applications.
 
 EXPOSE 8080
 
 ENV NAME=ruby \
     RUBY_VERSION=2.4 \
-    VERSION=0 \
-    RELEASE=1 \
-    ARCH=x86_64
+    RUBY_SHORT_VER=24 \
```

Comment:
Note that Ruby in Fedora 28 was upgraded to version `2.5.3`. Ruby 2.4 is in Fedora 27.

Reply:
Scratch that, I've read the Fedora version from the other Dockerfile.